android/java/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/constants/MyLocationTracking.java

```diff
@@ -0,0 +1,43 @@
+package com.mapbox.mapboxsdk.constants;
+
+import android.support.annotation.IntDef;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+
+public class MyLocationTracking {
```

Comment:
You should add some Javadoc for the class itself, see the Styles class for an example.